official/resnet/resnet_run_loop.py

```diff
@@ -588,7 +591,7 @@ def input_fn_eval():
           shape, batch_size=flags_obj.batch_size, dtype=export_dtype)
     classifier.export_savedmodel(flags_obj.export_dir, input_receiver_fn,
                                  strip_default_attrs=True)
-
+  return eval_results
```

Comment:
btw, what is the benchmark_logger thing from before used for ?

Reply:
I plan on removing it as it was used as part of the previous benchmark system.  I did not want to remove it until we were back up and running.  The old system some what counted on you using the logger and my approach is `return stats` or some other method where the test gets access to the results without having to check a log file.